lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -159,23 +159,33 @@ def find_urls(course_id, block, child_to_parent, request):
     return unit_url, section_url
 
 
-def video_summary(course, course_id, video_descriptor, request, local_cache):
+def video_summary(video_profiles, course_id, video_descriptor, request, local_cache):
     """
     returns summary dict for the given video module
     """
-    # First try to check VAL for the URLs we want.
-    val_video_info = local_cache['course_videos'].get(video_descriptor.edx_video_id, {})
-    if val_video_info:
-        video_url = val_video_info['url']
+    # Get encoded videos
+    video_data = local_cache['course_videos'].get(video_descriptor.edx_video_id, {})
+
+    # Get highest priority video to populate backwards compatible field
+    default_encoded_video = {}
+
+    if video_data:
+        for profile in video_profiles:
+            if video_data['profiles'].get(profile):
+                default_encoded_video = video_data['profiles'].get(profile)
+                break
```

Comment:
Complete nitpick.
Instead of duplicated code, this can be rewritten as:
```
default_encoded_video = video_data['profiles'].get(profile)
if default_encoded_video:
  break
```